samples/snippets/fsharp/tour.fs

```diff
@@ -842,9 +842,9 @@ module UnitsOfMeasure =
 /// Classes are a way of defining new object types in F#, and support standard Object-oriented constructs.
 /// They can have a variety of members (methods, properties, events, etc.)
 ///
-/// To learn more about Classes, see: https://docs.microsoft.com/dotnet/articles/fsharp/language-reference/classes
+/// To learn more about Classes, see: https://docs.microsoft.com/dotnet/fsharp/language-reference/classes
 ///
-/// To learn more about Members, see: https://docs.microsoft.com/dotnet/articles/fsharp/language-reference/members
+/// To learn more about Members, see: https://docs.microsoft.com/dotnet/fsharp/language-reference/members
```

Comment:
This link is not working for me. Maybe it got deleted and a redirect not put in place. I'll focus on just removing articles on this PR.